core/survey/_survey/surveyInfo.js

```diff
@@ -71,6 +72,8 @@ export const getLabel = (surveyInfo, lang) => {
   return StringUtils.isBlank(label) ? getName(surveyInfo) : label
 }
 
+export const getSamplingPolygon = ObjectUtils.getProp(keys.samplingPolygon, { value: '{}' })
```

Comment:
should this be by default an empty object or an object with a property "value" like you have defined here?

Reply:
Correct, forgot to change this from the first version that worked with value